### PR TITLE
WIP Persian translation

### PR DESCRIPTION
**Pull request by PouRiaBG**
- [X] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [X] Have you explained what your changes do, and why they add value to the Guides?
 

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----
This PR is a WIP to translate opensource.guide to Persian.

**Articles translation completed**

- [ ] Best Practices 

- [ ] Building Community

- [ ] Code of Conduct

- [ ] Finding Users

- [ ] Getting Paid

- [ ] How to Contribute [WIP]

- [ ] Leadership and Governance

- [ ] Legal

- [ ] Metrics

- [ ] Starting a project

### CODE REVIEW

**Comment by kittysnuggles** [comment]
https://github.com/github/opensource.guide/pull/1541#r38871511

**Comment by 666339** [comment]
Nature complexion on YouTube

File: _data/locales/fa.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+fa:
+  locale_name: فارسی
+  nav:
+    about: درباره
+    contribute: مشارکت کنید
+  index:
+    lead: نرم‌افزارهای متن‌باز توسط افرادی مثل شما ساخته‌شده است. بیاموزید چگونه پروژه خود را راه‌اندازی کنید و رشد دهید.
+    opensourcefriday: امروز جمعه است! چند ساعتی را برای مشارکت در نرم‌افزاری که استفاده می‌کنید و دوست دارید سرمایه‌گذاری کنید!
```
Comment (7playpersian) [inline _data/locales/fa.yml:8]:
```suggestion
    opensourcefriday: امروز جمعه است! چند ساعتی را برای مشارکت در نرم‌افزاری که استفاده می‌کنید و دوست دارید سرمایه‌گذاری کنید!
```

**Comment by marknguyen92** [comment]
Hello

**Comment by lkamazaezuzemedium** [comment]
Git add

**Comment by MikeMcQuaid** [comment]
Closing this as it's inactive and has failing CI. Sorry!

**Pull request closed without merging**